.github/workflows/collector-generate-and-update.yml

```diff
@@ -23,50 +19,14 @@ jobs:
         with:
           python-version: '3.12.6'
 
-      - name: Install dependencies
-        run: |
-          python -m pip install --upgrade pip
-          pip install -r requirements.txt
-
-      - name: Run update task
-        env: 
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: inv -e collector.update
+      - name: Set up Go
+        uses: actions/setup-go@41dfa10bad2bb2ae585af6ee5bb4d7d973ad74ed # v5.1.0
+        with:
+          go-version: '1.22.8'
 
-      - name: Run generate task
+      - name: Run Collector Update Bash Script
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: inv -e collector.generate
-
-      - name: Check for changes
-        id: check_changes
-        run: |
-          git config --global user.name "github-actions[bot]"
-          git config --global user.email "github-actions[bot]@users.noreply.github.com"
-          git add .
-          if git diff-index --quiet HEAD; then
-            echo "No changes detected"
-            echo "changes_detected=false" >> $GITHUB_OUTPUT
-          else
-            echo "Changes detected"
-            echo "changes_detected=true" >> $GITHUB_OUTPUT
-          fi
-
-      - name: Commit changes
-        if: steps.check_changes.outputs.changes_detected == 'true'
-        run: |
-          git switch -c update-otel-collector-dependencies-${{ steps.date.outputs.date }}
-          git commit -m "Update OTel Collector dependencies and generate OTel Agent"
-          git push -u origin update-otel-collector-dependencies-${{ steps.date.outputs.date }}
-
-      - name: Install GitHub CLI
-        if: steps.check_changes.outputs.changes_detected == 'true'
-        run: |
-          sudo apt-get update
-          sudo apt-get install gh
-
-      - name: Create draft pull request
-        if: steps.check_changes.outputs.changes_detected == 'true'
         run: |
-          gh auth login --with-token <<< ${{ secrets.GITHUB_TOKEN }}
-          gh pr create --title "Update OTel collector dependencies" --body "This PR updates the OTel Collector dependencies to the latest version. Please ensure that all tests pass before marking ready for review." --base main --head update-otel-collector-dependencies-${{ steps.date.outputs.date }} --draft
+          chmod +x ./.github/scripts/otel-collector-update-generate.sh
```

Comment:
Is it possible to save this file as executable in `git`, maybe?

Reply:
yes it already is, I will remove this line